pkg/auth/rbac.go

```diff
@@ -133,13 +159,14 @@ func (r *Rbac) Run(ctx context.Context) {
 			if !r.doRbac(&conn) {
 				switch msgType {
 				case MSG_TYPE_IPV4:
-					if err = xdpNotifyConnRstV4(&xdpHandlerKeyV4{Tuple: tupleV4}); err != nil {
-						log.Errorf("XdpHandlerUpdateV4 FAILED, err: %v", err)
+					filledData := make([]byte, TUPLE_LEN-IPV4_TUPLE_LENGTH)
```

Comment:
what does this do?

Reply:
the size of the key of `map_of_auth` is 34 bytes, while `tupleV4` is 12 bytes, so we need to fill it up to 34 bytes